docs/upgrade.html

```diff
@@ -65,6 +65,8 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
                         </li>
                         <li>The function <code>onNewBatch</code> in <code>org.apache.kafka.clients.producer.Partitioner</code> class was removed.
                         </li>
+                        <li>The <code>control.plane.listener.name</code> config was removed use <code>controller.listener.names</code> instead.
```

Comment:
It seems controller.listener.names doesn't directly replace control.plane.listener.name since they have different functionalities. Perhaps it's more accurate to say controller.listener.names has been removed.

Reply:
Done. Thanks for review.